ansq/tcp/types/connection.py

```diff
@@ -21,73 +64,63 @@ def __init__(
         host: str = "localhost",
         port: int = 4150,
         *,
-        message_queue: asyncio.Queue = None,
-        on_message: Callable = None,
-        on_exception: Callable = None,
-        loop: AbstractEventLoop = None,
-        auto_reconnect: bool = True,
-        heartbeat_interval: int = 30000,
-        feature_negotiation: bool = True,
-        tls_v1: bool = False,
-        snappy: bool = False,
-        deflate: bool = False,
-        deflate_level: int = 6,
-        sample_rate: int = 0,
-        debug: bool = False,
-        logger: logging.Logger = None,
+        connection_options: ConnectionOptions = ConnectionOptions(),
+        **kwargs: Mapping[str, Any],
     ):
         from ansq.tcp.protocol import Reader
         from ansq.tcp.types import ConnectionStatus
         from ansq.utils import get_logger
 
+        if kwargs:
+            warnings.warn(
+                message=(
+                    "Passing connection options to `TCPConnection` using keyword "
```

Comment:
```suggestion
                    f"Passing connection options to `{type(self).__name__}` using keyword "
```

`TCPConnection` is the base class.

Reply:
Addressed in https://github.com/lig/ansq/commit/a2d462e68e3294aecbd70094603a303ae46f2e0d